src/ui/qgsnewhttpconnectionbase.ui

```diff
@@ -140,44 +140,44 @@
          <string>WMS/WMTS Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
-         <item row="5" column="0" colspan="2">
-          <widget class="QCheckBox" name="cbxWmsIgnoreAxisOrientation">
+         <item row="13" column="0" colspan="2">
+          <widget class="QCheckBox" name="cbxSmoothPixmapTransform">
            <property name="text">
-            <string>Ignore axis orientation (WMS 1.3/WMTS)</string>
+            <string>Smooth pixmap transform</string>
            </property>
           </widget>
          </item>
-         <item row="3" column="0" colspan="2">
-          <widget class="QCheckBox" name="cbxIgnoreGetFeatureInfoURI">
+         <item row="8" column="0" colspan="2">
+          <widget class="QCheckBox" name="cbxWmsIgnoreAxisOrientation">
            <property name="text">
-            <string>Ignore GetFeatureInfo URI reported in capabilities</string>
+            <string>Ignore axis orientation (WMS 1.3/WMTS)</string>
            </property>
           </widget>
          </item>
-         <item row="2" column="0" colspan="2">
-          <widget class="QCheckBox" name="cbxIgnoreGetMapURI">
+         <item row="1" column="1">
+          <widget class="QComboBox" name="cmbTilePixelRatio"/>
+         </item>
+         <item row="2" column="0">
+          <widget class="QLabel" name="lblDefaultFeatureCount">
            <property name="text">
-            <string>Ignore GetMap/GetTile/GetLegendGraphic URI reported in capabilities</string>
+            <string>Default maximum number of GetFeatureInfo results</string>
```

Comment:
```suggestion
            <string>Maximum number of GetFeatureInfo results</string>
```

Reply:
ok "Default" removed, it was to make it clear that this is a connection-level setting that can be overridden when adding the layers from the datasource manager but I guess it is more intuitive this way.